apps/server/src/utils/getFileListFromFolder.ts

```diff
@@ -1,11 +1,28 @@
 import { ProjectFile } from 'ontime-types';
+import { getAppDataPath } from '../setup.js';
 
 import { readdirSync, statSync } from 'fs';
+import { extname, join } from 'path';
 
-export const getFileListFromFolder = (folderPath: string): Array<ProjectFile> => {
-  const files = readdirSync(folderPath);
-  return files.map((file) => {
-    const filePath = `${folderPath}/${file}`;
+const getFilesFromFolder = (folderPath: string): Array<string> => {
+  return readdirSync(folderPath);
+};
+
+const filterProjectFiles = (files: Array<string>): Array<string> => {
+  return files.filter((file) => {
+    const ext = extname(file).toLowerCase();
+    return ext === '.json';
+  });
+};
+
+export const getProjectFiles = (): Array<ProjectFile> => {
```

Comment:
I am not sure we should use the synchronous versions of the functions here

We did in reading files, since we are about to change the source of data of the app. Meaning that any other requests would have its source of truth invalidated by this operation

This is not the case here. We are just getting a list of files and will likely not want to block the process with this